packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -497,6 +513,59 @@ export class JobService {
     },
   };
 
+  private createCvatActions: Record<JobRequestType, CvatAction> = {
+    [JobRequestType.HCAPTCHA]: {
+      getElementsCount: async () => 0,
+    },
+    [JobRequestType.FORTUNE]: {
+      getElementsCount: async () => 0,
+    },
+    [JobRequestType.IMAGE_BOXES]: {
+      getElementsCount: async (
+        requestType: JobRequestType,
+        data: CvatDataDto,
+      ) => (await listObjectsInBucket(data.dataset, requestType)).length,
+    },
+    [JobRequestType.IMAGE_POINTS]: {
+      getElementsCount: async (
+        requestType: JobRequestType,
+        data: CvatDataDto,
+      ) => (await listObjectsInBucket(data.dataset, requestType)).length,
+    },
+    [JobRequestType.IMAGE_BOXES_FROM_POINTS]: {
+      getElementsCount: async (
+        requestType: JobRequestType,
+        data: CvatDataDto,
+      ) => {
+        if (!data.points) {
+          throw new ConflictException(ErrorJob.DataNotExist);
+        }
+
+        return (
+          await this.storageService.download(
+            generateBucketUrl(data.points, requestType),
+          )
+        ).annotations.length;
```

Comment:
You can place this in a function and reuse it for IMAGE_BOXES_FROM_POINTS and IMAGE_SKELETONS_FROM_BOXES